nflow-rest-api-jax-rs/src/main/java/io/nflow/rest/v1/jaxrs/WorkflowInstanceResource.java

```diff
@@ -116,44 +129,61 @@ public Response updateWorkflowInstance(@ApiParam("Internal id for workflow insta
 
   @GET
   @Path("/id/{id}")
-  @ApiOperation(value = "Fetch a workflow instance", notes = "Fetch full state and action history of a single workflow instance.")
-  @ApiResponses({ @ApiResponse(code = 200, response = ListWorkflowInstanceResponse.class, message = "If instance was found"),
-    @ApiResponse(code = 404, message = "If instance was not found") })
-  @SuppressFBWarnings(value = "LEST_LOST_EXCEPTION_STACK_TRACE", justification = "The empty result exception contains no useful information")
-  public Response fetchWorkflowInstance(@ApiParam("Internal id for workflow instance") @PathParam("id") long id,
-      @QueryParam("include") @ApiParam(value = INCLUDE_PARAM_DESC, allowableValues = INCLUDE_PARAM_VALUES, allowMultiple = true) String include,
-      @QueryParam("maxActions") @ApiParam("Maximum number of actions returned for each workflow instance") Long maxActions,
-      @QueryParam("queryArchive") @ApiParam("Query also the archive if not found from main tables") Boolean queryArchive) {
-    return handleExceptions(
-        () -> ok(super.fetchWorkflowInstance(id, include, maxActions, ofNullable(queryArchive).orElse(QUERY_ARCHIVED_DEFAULT), workflowInstances, listWorkflowConverter)));
+  @Operation(summary = "Fetch a workflow instance",
+      description = "Fetch full state and action history of a single workflow instance.")
+  @SuppressFBWarnings(value = "LEST_LOST_EXCEPTION_STACK_TRACE",
+      justification = "The empty result exception contains no useful information")
+  @ApiResponses({
+      @ApiResponse(responseCode = "200", description = "",
+          content = @Content(schema = @Schema(implementation = ListWorkflowInstanceResponse.class))),
+      @ApiResponse(responseCode = "404",
+          description = "If instance could not be created, for example when state variable value was too long") })
+  public Response fetchWorkflowInstance(@Parameter(description = "Internal id for workflow instance") @PathParam("id") long id,
+      @QueryParam("include") @Parameter(description = INCLUDE_PARAM_DESC) String include,
```

Comment:
Pitäiskö tämä muuttaa tästä "include=include1,include,include3" formaatista muotoon "include=include1&include=include2&include=include3" tms?

Reply:
Personally I think each include in separate parameter is more classic style. But some modern frameworks do not support it easily anymore.
So I would support both, but only document the multiple repeated parameters.
If we get `String[] include` as parameter we can do
```Stream.of(include).flatMap(i -> Stream.of(i.split(",")).collect(toSet())``` to flatten both cases into a set that can be later queried fast.

Reply:
Old format is deprecated but still supported by `include` param.
Added support for new format with `includes` param.